fishtank/src/cluster.ts

```diff
@@ -36,4 +36,19 @@ export class Cluster {
     })
     return new Node(this, name)
   }
+
+  async teardown(): Promise<void> {
+    // Remove containers
+    const removeContainers = []
```

Comment:
nit, the variable name can just be `containers`, since its getting all the containers in the cluster.